tests/Doctrine/Tests/LazyArrayCollection.php

```diff
@@ -10,13 +11,20 @@
  */
 class LazyArrayCollection extends AbstractLazyCollection
 {
+    private $_onInitialization;
```

Comment:
Docblock. Also, please don't prefix private/protected stuff with `_` (we used to do it with PHP 4, when visibility wasn't supported yet)